azkaban-db/src/main/sql/upgrade.3.42.0.to.3.43.0.sql

```diff
@@ -1,7 +1,13 @@
--- DB Migration from release 3.20.0 to 3.22.0
+-- DB Migration from release 3.42.0 to 3.43.0
+-- PR #1657 changes project cache to case insensitive.
 --
--- Release 3.21.0 is broken
```

Comment:
Why are these comments removed? Where did they come from?

Reply:
Previously I committed the wrong file by mistake and then fixed it. Sorry for the confusion. :)